quiche/src/recovery/gcongestion/bbr2/network_model.rs

```diff
@@ -555,7 +555,7 @@ impl BBRv2NetworkModel {
         true
     }
 
-    pub(super) fn is_inflight_too_high(
+    pub(super) fn is_loss_too_high(
```

Comment:
Please undo rename.  My understanding is that these names were chosen to be consistent with a C++ codebase that implements similar functionality.